cmd/unused-exporter/exporter.go

```diff
@@ -133,10 +146,24 @@ func (e *exporter) Collect(ch chan<- prometheus.Metric) {
 					labels[k] = meta[k]
 				}
 				e.logger.Log("unused disk found", labels)
-				countByNamespace[meta["kubernetes.io/created-for/pvc/namespace"]] += 1
+
+				if di, ok := diskInfoByNamespace[meta["kubernetes.io/created-for/pvc/namespace"]]; ok {
+					di.Count += 1
+					di.SizeByType[d.DiskType()] += int64(d.SizeGB())
+				} else {
+					diskInfoByNamespace[meta["kubernetes.io/created-for/pvc/namespace"]] = &namespaceInfo{
+						Count: 1,
+						SizeByType: map[unused.DiskType]int64{
+							d.DiskType(): int64(d.SizeGB()),
+						},
+					}
+				}
```

Comment:
Couple of nits, but I think it make the code more readable in the end. Let me know what you think:
```suggestion
				ns := meta["kubernetes.io/created-for/pvc/namespace"]
				di := diskInfoByNamespace[ns]
				if di == nil {
					di = &namespaceInfo{
						SizeByType: make(map[unused.DiskType]int64),
					}
					diskInfoByNamespace[ns] = di
				}
				
				di.Count += 1
				di.SizeByType[d.DiskType()] += int64(d.SizeGB())
```